server/static/js/dropzone.js

```diff
@@ -0,0 +1,78 @@
+function initDropzone(elem, token) {
+    Dropzone.options.myDropzone = {
+
+        // Prevents Dropzone from uploading dropped files immediately
+        autoProcessQueue: false,
+        uploadMultiple: true,
+        addRemoveLinks: true,
+        clickable: true,
+        parallelUploads: 200,
+        maxFiles: 200,
+        maxFilesize: 10, // MB
+        previewTemplate: document.getElementById('dz-preview-template').innerHTML,
+        headers: {
+            'X-CSRFToken': token
+        },
+        init: function() {
+            var submitButton = document.querySelector("#dzSubmit")
+            myDropzone = this; // closure
```

Comment:
It's unclear why this needed - it was in the example code. I suspect it's so things outside the closure can reference the dropzone.